print/libpaper/Portfile

```diff
@@ -1,55 +1,62 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem      1.0
+PortGroup       github 1.0
+
+github.setup    rrthomas libpaper 2.1.2 v
+revision        0
+checksums       rmd160  6b3898d28155228927fa78f55bc747fbcd8e0bc2 \
+                sha256  1fda0cf64efa46b9684a4ccc17df4386c4cc83254805419222c064bf62ea001f \
+                size    1264200
+
+github.tarball_from  releases
 
-name            libpaper
-set mainversion 1.1.29
-# Keep in case need again in the future
-#set nmuversion  5
-#set realversion ${mainversion}+nmu${nmuversion}
-#version         ${mainversion}.${nmuversion}
-set realversion ${mainversion}
-version         ${mainversion}
 categories      print
-license         GPL-2
+# libpaper is LGPLv2.1+, paperspecs is public domain
+license         LGPL-2.1+ public-domain
 maintainers     nomaintainer
-platforms       darwin
 description     A library providing routines for paper size management
 long_description \
                 The paper library and accompanying files are intended \
                 to provide a simple way for applications to take \
                 actions based on a system- or user-specified paper size.
 
-homepage        https://packages.qa.debian.org/libp/libpaper.html
-master_sites    debian:libp/${name}/
-
-distname        ${name}_${realversion}
-worksrcdir      ${name}-${realversion}
-
-checksums       rmd160  8ab9f62014375cc307d3b3a21b813af030032e11 \
-                sha256  26330e21e9a3124658d515fd850b0cde546ff42d89b2596a5264c5f1677f0547 \
-                size    44942
-
-post-patch {
-    set fd [open [file join ${worksrcpath}/lib paperspecs] a 0644]
-    # additional paper sizes
-    foreach {n x y} {jisb0 2920 4127
-                     jisb1 2064 2920
-                     jisb2 1460 2064
-                     jisb3 1032 1460
-                     jisb4  729 1032
-                     jisb5  516  729
-                     jisb6  363  516} {
-        puts ${fd} "${n} ${x} ${y}"
-    }
-    close ${fd}
+subport libpaper-utils {
+    # We introduce a subport for files with different licenses.
+    # Including them in the 'libpaper' port would prevent the
+    # distribution of MacPorts binary archives for some prominent
+    # tools like Ghostscript or ImagicMagick.
+    description       The executables of the libpaper package
+    long_description  Provide the new 'paper' and deprecated 'paperconf' \
+                      utility programs that come with the libpaper \
+                      library. Both are used to print paper size \
+                      information.
+    # `paper` is GPL-3+, `paperconf` is GPL-2
+    license           GPL-3+ GPL-2
 }
 
 use_autoreconf  yes
 
-configure.args  --mandir=${prefix}/share/man    \
-                --sysconfdir=${prefix}/etc
+configure.args  --enable-relocatable
```

Comment:
```suggestion
configure.args  --enable-relocatable \
                --disable-silent-rules
```

Other MacPorts maintainers prefer verbose build/disabling silent rules, because showing the full commands in the build log like this:

```
ccache /usr/bin/clang -DHAVE_CONFIG_H -I. -I..  -DINSTALLDIR=\"/opt/local/bin\" -I../src/libgnu -I../src/libgnu -I../lib -I../lib -DREAL_INSTALLPREFIX="\"/opt/local\"" -I/opt/local/include -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk  -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64 -MT paper-paper.o -MD -MP -MF .deps/paper-paper.Tpo -c -o paper-paper.o `test -f 'paper.c' || echo './'`paper.c
depbase=`echo paperconf.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
	ccache /usr/bin/clang -DHAVE_CONFIG_H -I. -I..  -I../src/libgnu -I../src/libgnu -I../lib -I../lib -DREAL_INSTALLPREFIX="\"/opt/local\"" -I/opt/local/include -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk  -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64 -MT paperconf.o -MD -MP -MF $depbase.Tpo -c -o paperconf.o paperconf.c &&\
	mv -f $depbase.Tpo $depbase.Po
mv -f .deps/paper-paper.Tpo .deps/paper-paper.Po
/bin/sh .././build-aux//libtool-reloc /bin/sh ../libtool  --tag=CC   --mode=link ccache /usr/bin/clang  -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64 `".././build-aux//reloc-ldflags" "x86_64-apple-darwin21.6.0" "/opt/local/lib" /opt/local/bin` -L/opt/local/lib -Wl,-headerpad_max_install_names -Wl,-syslibroot,/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64 -o paper paper-paper.o ../lib/libpaper.la ../src/libgnu/libgnupaper.la     
/bin/sh .././build-aux//libtool-reloc /bin/sh ../libtool  --tag=CC   --mode=link ccache /usr/bin/clang  -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64  -L/opt/local/lib -Wl,-headerpad_max_install_names -Wl,-syslibroot,/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64 -o paperconf paperconf.o ../lib/libpaper.la ../src/libgnu/libgnupaper.la     
libtool: link: ccache /usr/bin/clang -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64 -Wl,-headerpad_max_install_names -Wl,-syslibroot -Wl,/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64 -o .libs/paperconf paperconf.o  -L/opt/local/lib ../lib/.libs/libpaper.dylib ../src/libgnu/.libs/libgnupaper.a
libtool: link: ccache /usr/bin/clang -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64 -Wl,-rpath -Wl,@loader_path/../lib -Wl,-headerpad_max_install_names -Wl,-syslibroot -Wl,/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -arch x86_64 -o .libs/paper paper-paper.o  -L/opt/local/lib ../lib/.libs/libpaper.dylib ../src/libgnu/.libs/libgnupaper.a
```

is sometimes more useful for troubleshooting than this:

```
  CC       paper-paper.o
  CC       paperconf.o
  CCLD     paper
```

Reply:
Done, thanks!  I've also rebased to master.